### PR TITLE
Add DC-SBP-CaaSP4-SAP-Data-Hub-2

### PR DESCRIPTION
**Pull request by sknorr**
A squashed version of Ulrich's PR, so far with the following fixes:

* syntax fix in the CaaSP3-Data Hub metadata file
* optipng'd the images

Remaining issues:
* ceph-health.png is missing entirely
* the replacement of `scc-sle.png` with `scc-sles.png` is creating issues with the earlier CaaSP 3-Data Hub paper. Weirdly, though, the embeds look like this and not the other way around:

  ```
  adoc/CaaSP40_DI3X_Install_Guide.adoc:690:image::scc-sle.png[scc-sle,480,640]
  adoc/CaaSP4_DH2X_Install_Guide.adoc:690:image::scc-sle.png[scc-sle,480,640]
  adoc/CaaSP_DH2X_Install_Guide.adoc:718:image::scc-sle.png[title="SUSE Customer Center Products",scaledwidth=80%]
  adoc/SAP-DI-30-CaaSP42-Install-Guide.adoc:463: image::scc-sles.png[scc-sle,480,640]
  ```
  I.e. the new papers reference the old file name but the old paper has been updated with the new file name.
* There are ~200 new images (around 20 MB total). That seems excessive for one/two papers.
* scc-sles.png and scc-ses.png seem like they were JPGs originally (given the artefacts). They were converted to fairly monstrous PNGs here -- DAPS does work with JPGs too, so it might be best to convert them back.

### CODE REVIEW

File: adoc/CaaSP40_DI3X_Install_Guide-docinfo.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	</mediaobject>
+	</cover>
+
+<date><?dbtimestamp format="B d, Y" ?></date>
+
+
+<abstract>
+
+	<para>
+SAP Data Hub 2 is the tool set to govern big amount of data. SUSE CaaS Platform 4 is the kubernetes base that makes deploying SAP Data Hub 2 easy. This document describes the installation and configuration of SUSE CaaS Platform 4 and SAP Data Hub 2.
```
Comment (sknorr) [inline adoc/CaaSP40_DI3X_Install_Guide-docinfo.xml:34]: The abstract for this and for DH2X is the same.

**Comment by chabowski** [comment]
@uschairer We still need to solve some open issues "together" ;-) ... 

You can commit the requested changes into this PR #153 from Stefan - you have got write access to the SBP repo to do so

1) Please commit file ceph-health.png - the image is completely missing
2) It seems that you use a lot of images that originally were JPGs - but were saved as PNGs later - and are really huge now. Seems a compression artifact has happened, which "blows up" the size of the image, but doesn't make the image better. Could you please check that, and if possible exchange the "wannabe PNG" files with the original JPG files?
3) Seems that your PR contained many images/screenshots that are not used in any of the three papers?Would you mind kicking those unused images out? 

As said, please commit into Stefan's PR - hope that works ... 

Thanks very much!
Meike

**Comment by chabowski** [comment]
@uschairer in the CaaSP3 paper, the image scc-sle.png got lost / is missing - can you add that, too? This is where the Travis check currently fails.

**Comment by chabowski** [comment]
@uschairer 
Now the images ceph-stage-0 ceph-stage-1 ceph-stage-2 ceph-stage-3 ceph-stage-4 salt-key are missing in 
DC-SBP-CaaSP4-SAP-Data-Hub-2

But - seems we are moving in the right direction with our goal of cleaning up the mess ;-) ...

**Pull request merged**